Source/Core/VideoCommon/VertexLoaderBase.h

```diff
@@ -77,11 +77,10 @@ class VertexLoaderBase
   int m_numLoadedVertices = 0;
 
 protected:
-  VertexLoaderBase(const TVtxDesc& vtx_desc, const VAT& vtx_attr);
-  void SetVAT(const VAT& vat);
+  VertexLoaderBase(const TVtxDesc& vtx_desc, const VAT& vtx_attr)
+      : m_VtxDesc{vtx_desc}, m_VtxAttr{vtx_attr} {};
 
   // GC vertex format
-  TVtxAttr m_VtxAttr;  // VAT decoded into easy format
+  VAT m_VtxAttr;
```

Comment:
Now easier format :)

Reply:
On a related note, I'm not sure if the comment below really makes sense anymore:

https://github.com/dolphin-emu/dolphin/blob/c350da023ded841c859e01ef6a38f32f7e4ea829/Source/Core/VideoCommon/VertexLoaderBase.h#L90-L91

I don't think there's any way to avoid use of `m_VtxDesc`; the comment only makes sense if the goal was to also include that info in TVtxAttr, in which case the comment doesn't apply anymore.

Reply:
Comment removed (and also, `m_VtxAttr` and `m_VtxDesc` are now changed to `const` in "Eliminate TVtxAttr" instead of "Move vertex size and component calculation to VertexLoaderBase").